### PR TITLE
roachprod: mount drives on c4 machines

### PR DESCRIPTION
**Pull request by andreimatei**
Release note: None

### CODE REVIEW

File: pkg/cmd/roachprod/vm/aws/support.go [hunk 1]
```diff
@@ -56,6 +57,14 @@ for d in $(ls /dev/nvme?n1); do
     echo "Disk ${d} already mounted, skipping..."
   fi
 done
+for d in $(ls /dev/xvdd); do
```
Comment (petermattis) [inline pkg/cmd/roachprod/vm/aws/support.go]: I think you can do something like `for d in $(ls /dev/{nvme?n1,xvdd})` and combine the two loops into one. If you want that to be slightly more explicit: `for d in $(ls /dev/nvume?n1 /dev/xvdd)`.

**Review by ajwerner — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/35525#-:-L_PAEyeCc-qfCayNx5v:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajwerner and @petermattis)

___
*[pkg/cmd/roachprod/vm/aws/support.go, line 60 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/35525#-L_P4j5knYKO6e0ZnbkB-r1-60:-L_PA7Mn0_rOex6DjXSC:b55a6jw) ([raw file](https://github.com/cockroachdb/cockroach/blob/1af57ee0954455cb8ab6a82e0c4445d3ca1b7e97/pkg/cmd/roachprod/vm/aws/support.go#L60)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

I think you can do something like `for d in $(ls /dev/{nvme?n1,xvdd})` and combine the two loops into one. If you want that to be slightly more explicit: `for d in $(ls /dev/nvume?n1 /dev/xvdd)`.
</blockquote></details>

seconded, otherwise LGTM

**Review by ajwerner — COMMENTED** [review]
Though I guess that is sort of a meaningless LGTM given that comment is most of the diff

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/35525#-:-L_PAQdI7GyHAoA0YJvb:b3kn7r2)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajwerner and @petermattis)

**Review by andreimatei — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/35525)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @petermattis)

___
*[pkg/cmd/roachprod/vm/aws/support.go, line 60 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/35525#-L_P4j5knYKO6e0ZnbkB-r1-60:-L_UhXk84Mc_8ueG3flo:bb7f2mf) ([raw file](https://github.com/cockroachdb/cockroach/blob/1af57ee0954455cb8ab6a82e0c4445d3ca1b7e97/pkg/cmd/roachprod/vm/aws/support.go#L60)):*
<details><summary><i>Previously, ajwerner wrote…</i></summary><blockquote>

seconded, otherwise LGTM
</blockquote></details>

done

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/35525#-:-La76yMh3aqoJual5-nK:b-ydi9ay)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @petermattis)

**Pull request merged**